app/assets/stylesheets/-deprecated/_border.scss

```diff
@@ -1,33 +0,0 @@
-$border-color: $brand-border;
-
-// Borders
```

Comment:
This is a complete duplicate of Basscss's color borders file. Not sure why it was even here.